### PR TITLE
chore: new parameters for local pairing

### PR DESCRIPTION
**Pull request by igor-sirotin**
Closes https://github.com/status-im/status-desktop/issues/11435

Requires:
- https://github.com/status-im/status-go/pull/5643
- https://github.com/status-im/status-go/pull/5667

This PR finalizes the work done in:
- https://github.com/status-im/status-desktop/pull/14139 (released in 2.29)
- https://github.com/status-im/status-desktop/pull/15090 (released in 2.30)

# Description

1. Update `inputConnectionStringForBootstrapping` to the new API with `CreateAccount`

1. Delete `network_constants.nim`
This finally removes `NODE_CONFIG` and `NETWORKS` constants 🎉 

2. Remove `getDefaultNodeConfig` method

3. Make `buildCreateAccountRequest` and `buildWalletSecrets` free functions

4. Remove unused CLI options:
    - `WAKU_V2_PORT`
    - `STATUS_PORT`

### CODE REVIEW

**Review by saledjenic — APPROVED** [review]

**Review by jrainville — APPROVED** [review]
Nice work

File: src/app_service/service/accounts/service.nim [hunk 1]
```diff
@@ -225,16 +184,12 @@ QtObject:
...
     return CreateAccountRequest(
         rootDataDir: main_constants.STATUSGODIR,
         kdfIterations: KDF_ITERATIONS,
-        password: hashPassword(password),
-        displayName: displayName,
-        imagePath: imagePath,
-        imageCropRectangle: imageCropRectangle,
         customizationColor: DEFAULT_CUSTOMIZATION_COLOR,
-        emoji: self.defaultWalletEmoji,
+        emoji: "", # self.defaultWalletEmoji,
```
[the remark this answers is not in the document]
Comment (igor-sirotin) [reply src/app_service/service/accounts/service.nim:192]: oh, this is something I actually forgot to address. Will open an issue about this and update this part.
Thanks for noting!

[the remark this answers is not in the document]
Comment (igor-sirotin) [reply src/app_service/service/accounts/service.nim:192]: - https://github.com/status-im/status-go/issues/5663

[the remark this answers is not in the document]
Comment (igor-sirotin) [reply src/app_service/service/accounts/service.nim]: Implemented in status-go:
- https://github.com/status-im/status-go/pull/5667

Added corresponding changes in this PR: a38bbc2f4e56295bb18685c6928f106bd7bd8e0a

**Pull request merged**